textacy/spacier/doc_extensions.py

```diff
@@ -481,8 +482,14 @@ def to_bag_of_words(doc, normalize="lemma", weighting="count", as_strings=False)
             Note: The resulting set of frequencies won't (necessarily) sum
             to 1.0, since punctuation and stop words are filtered out after
             counts are normalized.
-        as_strings (bool): if True, words are returned as strings; if False
+        as_strings (bool): If True, words are returned as strings; if False
             (default), words are returned as their unique integer ids
+        remove_stop (bool): If True (default), stop words are removed after
+            counting.
+        remove_punct (bool): If True (default), punctuation tokens are removed
+            after counting.
+        remove_space (bool): If True (default), whitespace tokens are removed
```

Comment:
Just checking: Are whitespace tokens ever actually included? I thought spaCy filtered them out, but maybe I just never ran into a good example... Either way, I think we would _always_ want to exclude whitespace "words", which means we should take it out of the kwargs and instead just apply the logic under the hood.

Reply:
Good point! I hadn't really though about that properly. I've removed the "remove_space" flags as you suggested. Whitespace "words" are always removed.